builtin/credential/approle/path_role.go

```diff
@@ -163,6 +167,12 @@ TTL will be set to the value of this parameter.`,
 					Type:        framework.TypeString,
 					Description: "Identifier of the role. Defaults to a UUID.",
 				},
+				"enable_local_secret_ids": &framework.FieldSchema{
+					Type: framework.TypeBool,
+					Description: `
```

Comment:
Is the newline here intentional?

Reply:
Yep. I did it just so I could linewrap-format the description properly using `gq`.

Reply:
I removed it though as it was inconsistent with other descriptions. 👍